### DISCUSSION TITLE
Customizable icon order in the Taskbar / ribbon (on the Top)

### DISCUSSION THREAD

**Discussion by Rimos911** [nextcloud/all-in-one#5273] [Ideas] [answered]
Is it possible to imply a feature that the User can customize the order of the icons in the Taskbar / ribbon (on the Top). Currently the order is fix and not customizable. An drag and drop sulution would be nice (if the webinterface support this function)

**Comment by szaimen** [comment, marked as answer]
Hi, please open a thread for this at https://help.nextcloud.com. Thanks!
